docs/README.md

```diff
@@ -740,6 +741,43 @@ With proper concurrency logic in place, the latest run might have been cancelled
           workflow: my-workflow.yml
 ```
 
+### github-dependabot-check
+
+Action to check if a pull request is opened by dependabot with a specific label.
+
+```yaml
+      name: CI
+
+on:
+  push:
+
+jobs:
+  check-dependabot:
+    runs-on: ubuntu-latest
+    steps:
+      - name: dependabot check
+        id: dependabot
+        uses: ./.github/actions/github-dependabot-check
+        with:
+          gh-token: ${{ secrets.GITHUB_TOKEN }}
+          label: github_actions
+
+  deploy:
+    needs: check-dependabot
+    if: steps.dependabot.outputs.is_dependabot != 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy
+        run: echo "Deploying..."
+```
+How it works
+The dependabot check step uses your action to check if the PR is from Dependabot and has the github_actions label.
+If both conditions are true, the action sets an output (e.g., is_dependabot: true).
+The deploy job only runs if is_dependabot is not true, so deploys are skipped for these PRs.
+Benefit:
+This saves CI/CD resources and time by skipping unnecessary deploys for automated dependency updates that only affect GitHub Actions workflows.
```

Comment:
```suggestion
How it works
The dependabot check step uses your action to check if the PR is from Dependabot and has the github_actions label.
If both conditions are true, the action sets an output (e.g., result: true).
The deploy job only runs if result is not true, so deploys are skipped for these PRs.
Benefit:
This saves CI/CD resources and time by skipping unnecessary deploys for automated dependency updates that only affect GitHub Actions workflows.
```

Paragraph mentions `is_dependabot` instead of `result`. Also formatting is off and it shows all on one line, needs more new lines.

Reply:
I reviewed the text and fixed the markdown formatting that pre-commit was also complaining about - please review @mauriziovitale @dsibilio